dae/dae/utils/regions.py

```diff
@@ -25,6 +27,62 @@ def regions2bedfile(regions, bed_filename):
                 f"{reg.chrom}\t{reg.start-1}\t{reg.stop}\n")
 
 
+def split_into_regions(
+        chrom: str, chrom_length: int,
+        region_size: int) -> list[Region]:
+    """Return a list of regions for a chrom with a given length."""
+    regions = []
+
+    current_start = 1
+    while current_start < chrom_length + 1:
+        end = min(chrom_length, current_start + region_size - 1)
+        regions.append(Region(chrom, current_start, end))
+        current_start = current_start + region_size
+
+    return regions
+
+
+def get_chromosome_length(
```

Comment:
I would like to have `tabix` somewhere in the name of this function. Probaly `get_tabix_file_chromosome_length`, or `get_tabix_file_chrom_len` or something similar.